content/en/blog/_posts/2024-03-18-windows-ops-readiness.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "Introducing the Windows Operational Readiness Specification"
+date: 2024-03-18
+slug: intro-windows-ops-readiness
+---
+
+**Authors:** Jay Vyas (Tesla), Amim Knabben (Broadcom), and Tatenda Zifudzi (AWS)
+
+# Introducing the Windows Operational Readiness Specification
+
+Since Windows support [graduated to stable](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/release-notes-draft.md) with 1.14 in 2019, the capability to run Windows workloads has consistently been a major differentiator for Kubernetes distributions in the enterprise. However, with more Windows workloads being migrated to Kubernetes and new Windows features being continuously released, it became challenging to test Windows worker nodes in an effective and standardized way.
```

Comment:
Hmm. Old release notes are [broken](https://github.com/kubernetes/kubernetes/releases/tag/v1.14.0). I wonder if we can get SIG Release to fix that so we can link to the official release announcement? Or, link to https://kubernetes.io/blog/2019/03/25/kubernetes-1-14-release-announcement/ instead.

How about:
```suggestion
Since Windows support [graduated to stable](/blog/2019/03/25/kubernetes-1-14-release-announcement/)
with Kubernetes 1.14 in 2019, the capability to run Windows workloads has been much
appreciated by the end user community. The level of and availability of Windows workload
support has consistently been a major differentiator for Kubernetes distributions used by
large enterprises. However, with more Windows workloads being migrated to Kubernetes
and new Windows features being continuously released, it became challenging to test
Windows worker nodes in an effective and standardized way.
```